SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorCloudLauncherSettings.h

```diff
@@ -83,6 +107,32 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorCloudLauncherSettings : public UObje
 			: SnapshotPath.FilePath;
 	}
 
+	void SetPrimaryRegionCode(const ERegionCode::Type RegionCode);
+	FORCEINLINE FText GetPrimaryRegionCode() const
```

Comment:
Could this return an FString? if you use `GetEnumName` instead of `GetEnumTextByValue` ?https://api.unrealengine.com/INT/API/Runtime/CoreUObject/UObject/UEnum/GetEnumName/index.html

Reply:
The reason why it's text is so that I can easily use it here:
https://github.com/spatialos/UnrealGDK/blob/b1b277d689913065defceb985fb0c4369b0a441f/SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKSimulatedPlayerDeployment.cpp#L232

Why is the FString preferred (if it is)?

Reply:
I just thought most of the code made the functions look nicer, but text is actually the right way here as the text fields are used for localisation, so all UI text should be FText. Didn't know that, sorry! :)